uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/components/leave-request-popup-details-tab.component.js

```diff
@@ -437,6 +449,16 @@ define([
     }
 
     /**
+     * Gets called when the component is destroyed
+     */
+    function unsubscribeFromEvents () {
+      // destroy all the event
+      _.forEach(listeners, function (listener) {
+        listener();
```

Comment:
Does it destroy the listener?

Reply:
Yes :) Same is used in other components.